main.py

```diff
@@ -23,7 +23,14 @@ def launch_browser():
 def log_in(browser, asnb_username, asnb_password):
 
     logging.info('🔑 Logging in')
-    browser.find_element_by_class_name("btn-login").click()
+#     browser.find_element_by_class_name("btn-login").click()
```

Comment:
Can you remove the commented line here?

Reply:
Try added more delay time but the btn-login element still not interactable. If you side tested ok, please close the pull request.